src/AWS.Deploy.CLI/ServerMode/Controllers/DeploymentController.cs

```diff
@@ -128,6 +128,46 @@ public async Task<IActionResult> GetRecommendations(string sessionId)
             return Ok(output);
         }
 
+        /// <summary>
+        /// Gets a summary of the Recipe for the session's project.
+        /// </summary>
+        [HttpGet("session/<sessionId>/recipe/<recipeId>")]
+        [SwaggerOperation(OperationId = "GetRecipe")]
+        [SwaggerResponse(200, type: typeof(RecipeSummary))]
+        [Authorize]
+        public IActionResult GetRecipe(string sessionId, string recipeId)
```

Comment:
Seems like getting recipe metadata data has nothing to do with a deployment or an active session. I think we should add a separate ASP.NET Core controller for recipe metadata and the root path can me `recipe` instead of `session`. We don't need to make the methods require authentication or a session.

Reply:
Fixed